test/extensions/filters/http/aws_lambda/aws_lambda_filter_test.cc

```diff
@@ -417,7 +461,7 @@ TEST_F(AwsLambdaFilterTest, EncodeDataJsonModeStopIterationAndBuffer) {
  * headers while ignoring any HTTP/2 pseudo-headers.
  */
 TEST_F(AwsLambdaFilterTest, EncodeDataJsonModeTransformToHttp) {
-  setupFilter({Arn, false /*passthrough*/});
+  setupFilter({Arn, InvocationMode::Synchronous, false /*passthrough*/});
   filter_->resolveSettings();
   Http::TestResponseHeaderMapImpl headers;
   headers.setStatus(200);
```

Comment:
This seems to be repeated. Is it possible to reduce this code duplication?

Reply:
The only way I could think of is using parameterized tests.. :/